src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -175,17 +166,6 @@ class to the ``run`` method.
     6.2999999999... s [[-1.73302...e-08  1.31539...e-05  0.00000...e+00]] m
     """
 
-    # Some quantities are necessary for the particle tracker to function
-    # regardless of other configurations
-    _REQUIRED_QUANTITIES: typing.ClassVar[set[str]] = {
```

Comment:
Replaced this with a list that gets assembled by different setup methods, so it's more flexible for different configurations.